android/app/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,7 @@
       android:installLocation="auto"
       android:networkSecurityConfig="@xml/network_security_config"
       android:requestLegacyExternalStorage="true"
+      android:usesCleartextTraffic="true"
```

Comment:
👍 

for context https://developer.android.com/guide/topics/manifest/application-element#usesCleartextTraffic